flashpay/apps/account/serializers.py

```diff
@@ -4,9 +4,35 @@
 
 from rest_framework import serializers
 
+from flashpay.apps.account.models import APIKey
+from flashpay.apps.account.utils import generate_api_key
 from flashpay.apps.core.utils import decrypt_fernet_message
 
 
+class APIKeySerializer(serializers.ModelSerializer):
+    class Meta:
+        model = APIKey
+        fields = ("secret_key", "public_key", "network")
+
+
+class CreateAPIKeySerializer(APIKeySerializer):
+    class Meta(APIKeySerializer.Meta):
+        read_only_fields = ("secret_key", "public_key")
+
+    def validate(self, attrs: Any) -> Any:
+        network = attrs.get("network", "mainnet")
+        account = self.context["request"].user
+        # If API Key exists, the system deletes the keys
+        api_key = APIKey.objects.filter(account=account, network=network)
+        if api_key.exists():
+            api_key.delete()
```

Comment:
Why not try & except?
```suggestion
        try:
            api_key = APIKey.objects.get(account=account, network=network).delete()
        except APIKey.DoesNotExist:
            # do anything
```